test/e2e/helpers.js

```diff
@@ -289,6 +290,97 @@ const completeImportSRPOnboardingFlowWordByWord = async (
   await driver.clickElement('[data-testid="pin-extension-done"]');
 };
 
+const completeCreateNewWalletOnboardingFlow = async (
+  driver,
+  seedPhrase,
+  password,
+) => {
+  // welcome
+  await driver.clickElement('[data-testid="onboarding-create-wallet"]');
+
+  // metrics
+  await driver.clickElement('[data-testid="metametrics-no-thanks"]');
+
+  // create password
+  await driver.fill('[data-testid="create-password-new"]', password);
+  await driver.fill('[data-testid="create-password-confirm"]', password);
+  await driver.clickElement('[data-testid="create-password-terms"]');
+  await driver.clickElement('[data-testid="create-password-wallet"]');
+
+  // secure my wallet
+  await driver.clickElement('[data-testid="secure-wallet-recommended"]');
+
+  // reveal SRP
+  await driver.clickElement('[data-testid="recovery-phrase-reveal"]');
+
+  const revealedSeedPhrase = await driver.findElement(
+    '[data-testid="recovery-phrase-chips"]',
+  );
+
+  let recoveryPhrase = seedPhrase;
+
+  recoveryPhrase = await revealedSeedPhrase.getText();
+
+  await driver.clickElement('[data-testid="recovery-phrase-next"]');
+
+  // confirm SRP
+  const words = recoveryPhrase.split(/\s*(?:[0-9)]+|\n|\.|^$|$)\s*/u);
+  const finalWords = words.filter((str) => str !== '');
+  assert.equal(finalWords.length, 12);
+
+  await driver.fill('[data-testid="recovery-phrase-input-2"]', finalWords[2]);
+  await driver.fill('[data-testid="recovery-phrase-input-3"]', finalWords[3]);
+  await driver.fill('[data-testid="recovery-phrase-input-7"]', finalWords[7]);
+
+  await driver.clickElement('[data-testid="confirm-recovery-phrase"]');
+
+  await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+  // complete
+  await driver.findElement({ text: 'Wallet creation successful', tag: 'h2' });
+  await driver.clickElement('[data-testid="onboarding-complete-done"]');
+
+  // pin extension
+  await driver.clickElement('[data-testid="pin-extension-next"]');
+  await driver.clickElement('[data-testid="pin-extension-done"]');
+};
+
+const importWrongSRPOnboardingFlow = async (driver, seedPhrase) => {
+  // welcome
+  await driver.clickElement('[data-testid="onboarding-import-wallet"]');
+
+  // metrics
+  await driver.clickElement('[data-testid="metametrics-no-thanks"]');
+
+  // import with recovery phrase
+  await driver.pasteIntoField(
+    '[data-testid="import-srp__srp-word-0"]',
+    seedPhrase,
+  );
+
+  const warningText = 'Invalid Secret Recovery Phrase';
+  const warnings = await driver.findElements('.actionable-message__message');
+  const warning = warnings[1];
+
+  assert.equal(await warning.getText(), warningText);
+};
+
+const selectDropdownByNum = async (elements, index) => {
+  await elements[index].click();
+};
+
+const testDropdownIterations = async (options, driver, iterations) => {
```

Comment:
Optional: make the method name more precise such as `testSRPDropdownIterations`, as this method is in the file `helpers.js` which grows fast, a name that can explain itself will be easy to find and easy to be understood.

Reply:
I agree with you.